src/components/Header/Header.test.tsx

```diff
@@ -1,12 +1,29 @@
+import * as React from "react";
+import Cookies from "js-cookie";
 import { render, screen, waitFor, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
-import * as React from "react";
 import renderer from "react-test-renderer";
 
 import Header from "./Header";
 import { refineryResponse } from "./components/SitewideAlerts.test";
 
+const patronResponse = {
+  data: {
+    patron: {
+      barCodes: ["01234567890123"],
+      emails: ["test@nypl.org"],
+      id: "0123456",
+      names: ["PATRON, JANE"],
+    },
+  },
+  count: 1,
+  debugInfo: [],
+  statusCode: 200,
+};
+
+const mockCookieValue = '{"token_type":"Bearer", "access_token":"123abc"}';
```

Comment:
Let's also verify that we are calling the correct key in the browser's cookie. I'll describe it below in the `"Patron is logged in"` describe block.